src/oops/SOLID/singleResponsibilityPrinciple/before/SaveEmployeesMain.java

```diff
@@ -9,8 +9,12 @@ public static void main(String[] args) {
         List<Employee> employees = repository.findAll();
 
         // Save all
-        for (Employee e : employees){
-            Employee.save(e);
+        for (Employee emp : employees){
+            try {
+                repository.save(emp);
+            } catch(Exception exc) {
```

Comment:
Catching too broad exception

Reply:
Isn't it okay to catch some broad exception at client side, and have detailed exceptions caught in the deep layers?